src/demo/DemoTestLoanService.java

```diff
@@ -2348,4 +2351,122 @@ private static LoanAccount updateExpectedDisbursementDateAndFirstRepaymentDate(L
 		return updatedLoanAccount;
 	}
 
+	/**
+	 * Tests updating a loan in order to set a settlement account on it. It works only with Loans having Account Linking
+	 * enabled
+	 * 
+	 * @throws MambuApiException
+	 */
+
+	public static void testPatchSetllementAccounts() throws MambuApiException {
+
+		methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println(methodName = "\nIn " + methodName);
+
+		LoanAccount loanAccountToBeUpdated = newAccount;
+
+		LoansService loanService = MambuAPIFactory.getLoanService();
+		String productTypeKey = loanAccountToBeUpdated.getProductTypeKey();
+
+		System.out.println("Obtaining the product type of the loan account");
+		LoanProduct loanProduct = loanService.getLoanProduct(productTypeKey);
+
+		if (loanProduct.isAccountLinkingEnabled() == false) {
+
+			System.out.println("WARNING: " + methodName
+					+ " PATCH can`t be ran agains a loan that doesn`t have account linking enabled");
```

Comment:
Missing "t" in "agains". Also, I believe this message can be updated, we don't PATCH.